aten/src/ATen/templates/TensorMethods.cpp

```diff
@@ -25,7 +25,7 @@ namespace at {
 // This is temporary typedef to enable Quantizer in aten native function API
 // we'll remove them when we are actually exposing Quantizer class
 // to frontend
-using ConstQuantizerPtr = const c10::intrusive_ptr<Quantizer>&;
+// using ConstQuantizerPtr = const c10::intrusive_ptr<Quantizer>&;
```

Comment:
should we delete this instead of commenting it out?

Reply:
😬